src/pages/security/dataless-notifications.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Dataless notifications
+description: Learn how webhook providers like Microsoft OneNote send webhooks notifications without passing confidential data and the trade-offs with this approach.
+--- 
+
+{% table %}
+---
+* **Complexity**
+* - High
```

Comment:
Is this from the provider or consumer perspective? This is super easy for the provider to implement because they've pushed everything to the consumer.

Reply:
Both. While the webhook notification itself is simpler, both provider and consumer ends up with more work to accomplish the whole task:

- consumer needs to make extra calls, distribute api keys to places, manage secrets being spread, and write more code :(
- provider gets more api callbacks, introduce performance issues, and all crap.

Reply:
that's why I kept high...

Perhaps in the future we need to show complexity for both sides (not have this conflated).